src/generator/AutoRest.Go/Model/MethodGo.cs

```diff
@@ -79,6 +79,78 @@ internal void Transform(CodeModelGo cmg)
         }
 
         public string MethodSignature => $"{Name}({MethodParametersSignature})";
+        
+        public string MethodParametersSignatureComplete
+        {
+            get
+            {     
+                var signature = new StringBuilder("(");
+                signature.Append(MethodParametersSignature);
+                if (!IsLongRunningOperation())
+                {
+                    if (MethodParametersSignature.Length > 0)
+                    {
+                        signature.Append( ", ");
+                    }
+                    signature.Append("cancel <-chan struct{}");
+                }
+                signature.Append(")");
+                return signature.ToString();
+            }
+        }
+
+        public string MethodReturnSignatureComplete
+        {
+            get
+            {
+                var signature = new StringBuilder("(<-chan ");
+                signature.Append((ListElement.ModelType as SequenceTypeGo).GetElement);
+                signature.Append(", <-chan error)");
+                return signature.ToString();
+            }
+        }
+
+        public string ParametersDocumentation
+        {
+            get
+            {
+                StringBuilder sb = new StringBuilder();
+                foreach (var parameter in LocalParameters)
```

Comment:
I don't see a new line being added into the documentation anywhere. I think it makes sense to have each parameter have its own line.